### DISCUSSION TITLE
not finding ddlc save for import both stable and unstable

### DISCUSSION THREAD

**Discussion by jm4686042** [Monika-After-Story/MonikaModDev#9162] [QA - tech support + general] [answered]
![e](https://user-images.githubusercontent.com/101479441/158035233-b3da07aa-db2c-4c6a-92f8-1a9010d4bdc4.jpg)

**Comment by ThePotatoGuy** [comment, marked as answer]
DDLC save data is in AppData, check if its there first because its unlikely that the check here is broken.
